internal/config.go

```diff
@@ -39,6 +39,8 @@ const (
 	PgSslModeSetting             = "PGSSLMODE"
 	TotalBgUploadedLimit         = "TOTAL_BG_UPLOADED_LIMIT"
 	NameStreamCreateCmd          = "WALG_STREAM_CREATE_COMMAND"
+	NameStreamRestoreCmd         = "WALG_STREAM_RESTORE_COMMAND"
+	NameLogApplyCmdPath          = "WALG_MYSQL_BINLOG_APPLY_COMMAND_PATH"
```

Comment:
May be WALG_LOG_APPLY_COMMAND is better name ?  It's more db neutral and may be suitable for mongodb (oplog)